templates/nfs-provisioner-deployment.yaml.j2

```diff
@@ -17,19 +17,23 @@ spec:
       serviceAccountName: nfs-client-provisioner
       containers:
         - name: nfs-client-provisioner
+{% if ansible_architecture == "ppc64le" %}
+          image: ibmcom/nfs-client-provisioner-ppc64le:latest
```

Comment:
How about creating a variable for nfs-client-provisioner image and use separate values for each architecture ?
The vars-ppc64le.yaml will have the ppc64le specific image and the default will be the quay image.

Reply:
@bpradipt So I like this, but would still like to provide a default if none is provided.

ppc64le workflow:

1) If the user provides a custom Image, then that specific image is used.
2) If the user does NOT provide the image, it defaults to `ibmcom/nfs-client-provisioner-ppc64le:latest`

x86_64 workflow (similar):

1) If the user provides a custom Image, then that specific image is used.
2) If the user does NOT provide the image, it defaults to `quay.io/external_storage/nfs-client-provisioner:latest`

So in general I like the idea of the user being able to specify an image, but I would also like to have a "reasonable default" if the user doesn't provide it

Reply:
Coming back to this, I've made some modifications to this file and I think I know what you're talking about bpradipt. The ability to specify image.

Right now you can do..

```
nfs:
  server: 192.168.7.77
  path: /export
```

I could see in the future doing


```
nfs:
  server: 192.168.7.77
  path: /export
  image: quay.io/external_storage/nfs-client-provisioner:latest
```

However, since this specifically uses the nfs-provisioner from the [kuberentes upstream incubator](https://github.com/kubernetes-incubator/external-storage/blob/master/nfs-client/README.md), other than ppc64le, when would someone provide a custom image for this?

In general I'm for having the ability to specify an image (even if it's a "hidden" feature)